.github/workflows/npm-publish.yml

```diff
@@ -1,32 +1,48 @@
-name: Node.js Package
+name: Publish Indice.Angular
 
 on:
   workflow_dispatch:
-  release:
-    types: [created]
 
 jobs:
   publish-npm:
     runs-on: ubuntu-latest
+
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
     steps:
       - uses: actions/checkout@v4
 
-      - run: |
+      - name: generate package-lock.json
+        run: |
           npm i --package-lock-only
-        env:
-          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
 
       - uses: actions/setup-node@v4
         with:
           cache: 'npm'
           cache-dependency-path: 'package-lock.json'
           check-latest: true
-          node-version: 20
+          node-version: '20.19.5'
```

Comment:
'20.19.5' I think we could have `20.19` that will get the latest minor. Not sure